charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -109,6 +109,9 @@ serviceAccount:
 storageClasses: []
 # Add StorageClass resources like:
 # - name: ebs-sc
+#   # annotation metadata
+#   annotations:
+#     storageclass.kubernetes.io/is-default-class: "true"
```

Comment:
Did we miss the "labels" field here? It's omitted on the storageclass.yaml.

Reply:
I handled it in the template. I can't think of a great reason to have a label vs and annotation, so I didn't include it in the sample. I can if required.

Reply:
That'd be nice. Documenting all fields in the example could save operators from reading the template, so I think it's worth doing.

Reply:
@ayberk I added the label example. Let me know if that looks ok.